src/shared-components/dialogModal/__snapshots__/test.tsx.snap

```diff
@@ -76,14 +76,6 @@ exports[`<DialogModal /> renders dialog modal with custom color 1`] = `
   transform: translateY(100%);
 }
 
-.emotion-0 p {
```

Comment:
Let's wrap the `modalBody` content in these tests with the new `Paragraph` component so that usage is canonical/more documented.

Reply:
ok Im good with that